rsyslog/init.sls

```diff
@@ -50,6 +51,33 @@ rsyslog_mmjsonparse:
     - require:
       - pkg: rsyslog
 
+{%- else %}
+rsyslog_updates:
```

Comment:
maybe a better name for this for `adiscon_ppa_repo`

Reply:
Yeah I'll change both to that!